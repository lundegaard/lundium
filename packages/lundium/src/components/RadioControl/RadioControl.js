import React from 'react';

import FormControl from '../FormControl';

const RadioControl = props => (
	<FormControl component="input" type="radio" {...props} />
);

export default RadioControl;
