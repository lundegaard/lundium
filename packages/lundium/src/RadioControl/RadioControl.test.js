import React from 'react';
import { shallow } from 'enzyme';

import RadioControl from './RadioControl';

describe('<RadioControl />', () => {
	const requiredProps = {
		name: 'radioControlField',
	};

	it('shallow', () => {
		const wrapper = shallow(<RadioControl {...requiredProps} />);
		expect(wrapper).toMatchSnapshot();
	});
});
