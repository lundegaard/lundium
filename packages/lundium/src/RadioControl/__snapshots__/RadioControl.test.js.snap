// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RadioControl /> shallow 1`] = `
<FormControl
  component="input"
  name="radioControlField"
  type="radio"
/>
`;
